exporters/auto/src/main/java/com/google/cloud/opentelemetry/auto/GoogleCloudMetricExporterFactory.java

```diff
@@ -3,18 +3,25 @@
 import com.google.auto.service.AutoService;
 import com.google.cloud.opentelemetry.metric.MetricExporter;
 import io.opentelemetry.javaagent.spi.exporter.MetricExporterFactory;
-
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashSet;
 import java.util.Properties;
+import java.util.Set;
 
 @AutoService(MetricExporterFactory.class)
 public class GoogleCloudMetricExporterFactory implements MetricExporterFactory {
-    @Override
-    public MetricExporter fromConfig(Properties _config) {
-        try {
-            return MetricExporter.createWithDefaultConfiguration();
-        } catch (IOException ex) {
-            throw new RuntimeException(ex);
-        }
+  @Override
+  public MetricExporter fromConfig(Properties _config) {
+    try {
+      return MetricExporter.createWithDefaultConfiguration();
+    } catch (IOException ex) {
+      throw new RuntimeException(ex);
     }
-}
\ No newline at end of file
+  }
+
+  @Override
+  public Set<String> getNames() {
+    return new HashSet<>(Arrays.asList("google_cloud", "google_cloud_metric"));
```

Comment:
nit: we should probably define these as constants shared between trace and metric exporters
Also, `cloud_monitoring` (and `cloud_trace` respectively) might be more appropriate naming, as these are the names of the products?